src/main/cpp/src/ftos_converter.cu

```diff
@@ -0,0 +1,1161 @@
+/* Not a contribution
+ * Changes made by NVIDIA CORPORATION & AFFILIATES enabling ryu or otherwise documented as
+ * NVIDIA-proprietary are not a contribution and subject to the following terms and conditions:
+ *
+ * SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+ *
+ * NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+ * property and proprietary rights in and to this material, related
+ * documentation and any modifications thereto. Any use, reproduction,
+ * disclosure or distribution of this material and related documentation
+ * without an express license agreement from NVIDIA CORPORATION or
+ * its affiliates is strictly prohibited.
+ */
```

Comment:
This drops the copyrights from the original source.  Also need to add a NOTICE file with attribution to ryu per ASL license. See spark-rapids NOTICE file as an example.

Reply:
This is still not resolved.  The ryu copyrights have been dropped from this file and this file is missing the ASLv2 license.  Also ideally there should be a note stating what original file in ryu this was derived from.

Reply:
Thanks, I missed that. Added the ryu copyrights and license now.